sycl/include/CL/sycl/queue.hpp

```diff
@@ -248,7 +248,8 @@ class __SYCL_EXPORT queue {
     event Event;
 
 #if __SYCL_USE_FALLBACK_ASSERT
-    if (!is_host()) {
+    if (!is_host() && get_backend() != backend::cuda &&
```

Comment:
Could you, please, add a `FIXME` or `TODO` comment saying when check for back-end should be removed?

Reply:
Good idea. I have added a comment to both backend checks.